core/dbt/cli/main.py

```diff
@@ -73,6 +76,14 @@ def cli(ctx, **kwargs):
     if flags.RECORD_TIMING_INFO:
         ctx.with_resource(profiler(enable=True, outfile=flags.RECORD_TIMING_INFO))
 
+    project_root = flags.PROJECT_DIR if flags.PROJECT_DIR else os.getcwd()
```

Comment:
Is the if/else statement necessary here given `flags.PROJECT_DIR` [defaults](https://github.com/dbt-labs/dbt-core/blob/chenyu/project_with_click/core/dbt/cli/params.py#L233) to `Path.cwd()` via the [default_project_dir](https://github.com/dbt-labs/dbt-core/blob/374fce4e5b86882250db79945daf0b5cd3419a68/core/dbt/cli/resolvers.py#L4) method? 

Also, is there a semantic difference between the `project_root` and `project_dir` variable names or can the naming be standardized here?

Reply:
+1 to both of these comments.

Reply:
@MichelleArk  good catch, I didn't dig in there.
I will also update the name